ROADMAP.md

```diff
@@ -1,42 +1,45 @@
 # ROADMAP
 
-## K8s Native
+## Plugin system
 
-Perses as a dashboard tooling should be Kubernetes native. That means being able to deploy Perses on K8s and being able
-to deploy dashboard alongside the associated applications.
+When this document was created (in March 2024), we mentioned that we were going to entirely review the plugin system.
 
-For that, we aim to provide:
+Indeed, this has been reviewed but not yet released. We are very close to providing a first version.
+We are still working on this as we are missing a few things, like documentation, the ability to use it in the explorer,
+and basically cleaning the old system.
 
-- an operator that will deploy Perses and the required CRDs. Work is going one in the
-  repository https://github.com/perses/perses-operator.
-- a helm chart that should help to install Perses with or without the operator on k8s. Work is available in the
-  repository: https://github.com/perses/helm-charts.
+You can follow this issue to get updates on where we stand: https://github.com/perses/perses/issues/1543
 
-## Plugin system
+### Plugin Versioning
+
+In the second version of the new plugin system, we are going to tackle the topic of how plugins can be versioned and
+upgraded.
 
-While we already have a plugin system, it has a couple of limitations:
+Discussion is already ongoing on this topic: https://github.com/perses/perses/discussions/1186
 
-- First, we are not able to externalize the plugins. So that means any plugin needs to be built in the upstream
-  application which is super limited as we won't be able to accept and maintain any kind of plugin the community / end
-  user would like.
-- Then today, it's really hard to understand how we can implement a plugin, and that shouldn't be the case at all.
+### More Data-sources to come
 
-So for the next 6 months, we are going to entirely review the plugin architecture.
+Once the new plugin system is released, we will unblock the possibility to support other data sources besides Prometheus
+and Tempo.
 
-Here is the issue if you want to follow this topic: https://github.com/perses/perses/issues/1543
+We are going to support Logs Datasource, starting with OpenSearch and Loki.
```

Comment:
It will be really cool to support `OpenSearch` as it is fully open source and [added to linux foundation last year](https://www.linuxfoundation.org/press/linux-foundation-announces-opensearch-software-foundation-to-foster-open-collaboration-in-search-and-analytics) :)

Reply:
yup that's the plan :)